### PR TITLE
SALTO-2606 support more zendesk types for missing references- trigger

### PR DESCRIPTION
**Pull request by Eldardar**
Adding missing references for brand, schedule and group values in Trigger instances

To Do:

- [x] Add UT
- [ ] Support more types
- [ ] Support list values

---

_Additional context for reviewer_
* Changed BRAND_NAME const to BRAND_TYPE_NAME
* Moved missing references strategies to a different file for more order

---
_Release Notes_: 
* Adding missing references for brand, schedule and group values in Trigger instances

---
_User Notifications_: 
* missing references will convert from string into Salto unresolved references

### CODE REVIEW

File: packages/zendesk-adapter/src/filters/field_references.ts [hunk 1]
```diff
@@ -35,6 +36,15 @@ const neighborContextFunc = (args: {
   ...args,
 })
 
+// TODO: Support in types with list values
+const NEIGHBOR_FIELD_TO_TYPE_NAMES: Record<string, string> = {
```
Comment (noamhammer) [inline packages/zendesk-adapter/src/filters/field_references.ts:40]: Can't you use the regular contextValueMapper here?

Comment (Eldardar) [reply packages/zendesk-adapter/src/filters/field_references.ts:40]: Discussed f2f

**Review by noamhammer — APPROVED** [review]

File: packages/zendesk-adapter/src/filters/references/missing_references.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
+/*
```
Comment (netama) [inline packages/zendesk-adapter/src/filters/references/missing_references.ts:1]: what's the purpose of the new folder?

Comment (netama) [inline packages/zendesk-adapter/src/filters/references/missing_references.ts:1]: @Eldardar what's the purpose of the new folder?

**Pull request merged**